sdk/storage/azure-storage-blobs/src/rest_client.cpp

```diff
@@ -377,7 +377,7 @@ namespace Azure { namespace Storage { namespace Blobs {
       request.SetHeader("Content-Length", std::to_string(requestBody.Length()));
       request.GetUrl().AppendQueryParameter("restype", "service");
       request.GetUrl().AppendQueryParameter("comp", "properties");
-      request.SetHeader("x-ms-version", "2021-12-02");
```

Comment:
I know this is machine generated code, but what is the point of having an ApiVersion constant in rest_client.hpp if it's going to be ignored in rest_client.cpp?

Reply:
Yeah, it's machine generated code. We want to use this constant in rest_client.cpp, but in autorest, referencing a variable is more complicated than referencing a literal.
On the other hand, this ApiVersion constant is used in convenience layer such as `Sas Version`